student-ambassador.html

```diff
@@ -53,7 +53,6 @@ <h1>STUDENT <b>AMBASSADOR</b></h1>
 
             <!--            For mobile view-->
             <div class="d-block d-md-none mt-5 py-4">
-                <p style="line-height: 1.0;"> <h1 style='font-size:65px;'> <span style='font-weight:bold;'>Join the</span> Family</h1> </p>
                 <p > <h2>STUDENT <b>AMBASSADOR</b></h2>
                 </p>
```

Comment:
We don't need this `<p>` tag right?

Reply:
Yes, ok then I will remove it @anjula-sack

Reply:
Done @anjula-sack

Reply:
Can you merge it? @anjula-sack